Dockerfile

```diff
@@ -1,30 +1,31 @@
 # 1) use alpine to generate a void environment
 FROM alpine:3.9 as stage0
-ARG REPOSITORY=https://alpha.de.repo.voidlinux.org/current
+ARG REPOSITORY=https://alpha.de.repo.voidlinux.org
```

Comment:
`REPOSITORY` is not the right var name anymore. rename to `MIRROR`?

Reply:
yeah, that seems fine.  I'll make the change and merge later tonight.